### DISCUSSION TITLE
Conte como foi a sua experiência participando deste projeto.

### DISCUSSION THREAD

**Discussion by morgannadev** [morgannadev/octocats_da_comunidade#1] [General]
Conte aqui nessa discussão como foi a sua participação neste projeto. Se você sentiu dificuldades e quais foram elas. E o que você achou do meetup ou de ter participado do Hacktoberfest. Se foi a sua primeira vez ou não. Enfim, conte aquilo que você se sentir a vontade.

**Comment by morgannadev** [comment]
Oi, eu sou a Morganna e adorei participar desse projeto criando uma personagem octocat com a minha carinha. E agora estou me divertindo fazendo parte do Hacktoberfest.

**Comment by Anemaygi** [comment]
Primeira vez que estou participando do hacktoberfest e estou assistindo a live agora :) Bem animada para participar e contribuir em diferentes projetos

**Reply by morgannadev** [reply]
Muito obrigada por ter participado da live e do nosso projeto! Arrasou! Que venham mais PRs incríveis para você continuar animadíssima!

**Comment by pamelars86** [comment]
Conheci o Hacktoberfest no 2021 contudo é a primeira vez que estou participando :) 
Gostaria continuar contribuindo em projetos Open Source e comunidade ❤️ , acredito fortemente que juntos podemos criar grandes coisas e gerar inovação💪

**Reply by morgannadev** [reply]
Boas vindas ao hacktoberfest, que bom que curtiu participar e que tá dando seu primeiro passo nesse evento incrível! E o espírito é esse mesmo, continuar participando de projetos open source! Obrigada por ter contribuído com o projeto!

**Comment by mechamogeo** [comment]
Não é a primeira vez que eu estou participando de um Hacktoberfest mas faz bastante tempo que eu não faço um :)
Gostaria de contribuir mais para os projetos da comunidade porém não tive boas experiências no passado por diretrizes de alguns projetos que acabaram me deixando pra baixo, pois ninguém aceitava as PR's e não explicavam o porque 💔 .

Estou na fase de estudar LLM's e mais sobre Inteligência artificial e gostaria de contribuir mais para esses projetos 👓

**Reply by morgannadev** [reply]
Infelizmente alguns projetos e pessoas mantenedoras acabam não dando retorno, né? Mas espero que você consiga participar de projetos mais amigáveis. Arrase nos PRs por aí! Muito obrigada por ter participado com a comunidade da nossa live e do nosso projeto!

**Comment by letyhashimoto** [comment]
Olá! Ainda estou um pouco perdida para fazer PRs oficiais do Hacktoberfest. Este ainda é meu primeiro! Espero que dê tudo certo e eu consiga meus badges só pra eu me sentir mais confiante em contribuir com projetos open source e ficar feliz por ter participado do meu primeiro Hacktoberfest! Já tinha visto os anteriores, mas não tive coragem suficiente para participar e nem sabia fazer PRs direito, me sentindo super insegura! Agradeço à Morgs e as comunidades Feministech, Github Community SP pelo apoio e helps para ser mais ativa nos códigos e na área tech! To adorando participar e quero conseguir contribuir mais! ❤️

**Reply by morgannadev** [reply]
Já está dando certo! Obrigada demais por ter participado do nosso meetup em live e também por ter vindo contribuir com o nosso projeto! Eu também me sentia da mesma forma. Mas aos poucos eu fui conseguindo participar cada vez mais. Tenho certeza de que você vai conseguir contribuir ainda mais! Que venham essas badges maravilhosas! Obrigada demais, Lety!

**Comment by julianaconde** [comment]
Primeira vez que estou participando do evento e acompanhando as discussões da comunidade. Estou achando incrível e com certeza participarei mais! Obrigada pela oportunidade de participar!

**Reply by morgannadev** [reply]
Boas vindas ao evento, então! Que bom que curtiu participar. Muito obrigada por estar na live com a gente e por ter contribuído com o nosso projeto! Espero que tenha sido divertido!

**Comment by julianajjo** [comment]
Olá! Essa é minha primeira participação na Hacktoberfest, estou ansiosa para contribuir com os projetos! Gostaria de agradecer ao GitHub e a Morganna por compartilhar o passo-a-passo para essa primeira contribuição.

**Reply by morgannadev** [reply]
Maravilhosa! Obrigada demais por ter participado da live e por ter contribuído com o nosso projeto! Arrasou! Adorei que já está participando do evento pela primeira vez. Que venham muito mais aprendizado e contribuições.

**Comment by mvteus** [comment]
Esse é a minha primeira contribuição para o Hacktoberfest, esse seria meu segundo PR, sempre tive o interesse em contribuir com o open source desde que conheci o Linux. Espero que esse seja apenas os primeiros de muitos PR, e que eu possa contribuir para as comunidades e tornar a tecnologia cada vez mais acessível.

**Reply by morgannadev** [reply]
O Linux também é um caminho lindo para o Open Source mesmo. Que bom que começou a contribuir. Espero que seja a porta de entrada para continuar participando sempre que puder!

Valeu demais por ter participado da live e por ter contribuído para o projeto!

**Comment by joaovictor1516** [comment]
Olá conheci o Hacktoberfest este ano e já estou amando e pretendo ser cada vez mais presente na comunidade opensorce, espero ter escrito certo kkkkk.

**Reply by morgannadev** [reply]
Oie! Que bom que conheceu este ano e já está participando, parabéns! E valeu demais por ter acompanhado a live e também participado do nosso projeto para treinar!

Sobre a palavra, seria "open source". Mas não tem problema. Bora participar cada vez mais!

**Comment by madalena-rocha** [comment]
Olá, sou Madalena e esta é minha segunda vez no Hacktoberfest. Minha experiência neste projeto foi a melhor possível, nunca pensei que contribuir para um projeto pudesse ser tão divertido! Além disso, o meetup foi uma ótima oportunidade para conhecer pessoas e projetos interessantes. Estou grata pela oportunidade de participar.

**Reply by morgannadev** [reply]
Muito obrigada, Madalena! Fico feliz que tenha achado divertido, porque também achei bem legal. Ainda mais criando octocats com a nossa carinha. Que bom que curtiu o meetup, obrigada por ter participado!

**Comment by priscillatrevizan** [comment]
Olá! Este é meu segundo ano participando da Hacktoberfest, mas o primeiro que me aventuro contribuindo em projetos. Minha experiênca neste projetp foi muito didática e positiva, e com certeza está destravando alguns receios e impedimentos no mundo open source!
Muito Grata por sua contribuição @morgannadev !

**Reply by morgannadev** [reply]
Tô bem feliz por você estar perdendo o medo e participando com a gente nesse nosso projeto. Eu também tinha bastante medo, mas a comunidade foi me ajudando e deu tudo certo. Que bom que achou didático, de verdade! Bora continuar contribuindo juntas!

**Comment by parlandin** [comment]
Hi! essa é a primeira vez participando do Hackatoberfest! Ter participado desse evento com a orientação @morgannadev foi uma experiência ótima, apesar de já ter uma certa experiência com github, ainda sim aprendi muito. Muito obrigado!

**Reply by morgannadev** [reply]
Que massa! Com certeza entender como o GitHub funciona já ajuda bastante. Obrigada por ter participado e contribuído com o nosso projeto!

**Comment by Gh3st** [comment]
eae essa é a primeira vez que eu to fazendo o hacktoberfest, espero aprender coisas novas com esse projeto

**Reply by morgannadev** [reply]
Fico feliz que esteja participando pela primeira vez e topando ajudar mais projetos, é sobre isso!

**Comment by sucodelarangela** [comment]
Olá, eu sou a **sucodelarangela** e sou Dev Front End. Já conhecia o **Hacktoberfest**, apesar de nunca ter participado efetivamente. Esta é minha primeira participação oficial, mentorada e orientada pela @morgannadev através de um evento na comunidade Orange Juice Tech, do Discord. Amei toda a experiência e pretendo participar mais vezes! Um xero!

**Reply by morgannadev** [reply]
Ahhhh, muito massa saber que tá participando pela primeira vez com a comunidade! Que seja a primeira de muitas! Obrigada por todo o apoio que deu no nosso evento, foi incrível contar com a sua participação e experiência também. Trouxe dicas valiosíssimas! Xeroooo!

**Comment by Danny-s07** [comment]
Olá, primeira vez participando do Hacktoberfest! achei achei incrivel participar, apanhando um pouco. Mas muito legal conhecer esses eventos de tecnologia, obrigada por apresentar esse evento @morgannadev e fazer uma explicação com toda a paciência pra gente la no Discord da Orange Juice Tech <3.

**Reply by morgannadev** [reply]
Tá tudo bem ter dificuldades. Estamos juntas nessa para nos ajudarmos. Os eventos e comunidades estão aí para serem um apoio mesmo! Fico feliz que tenha gostado tanto da explicação. Muito obrigada por participar junto com a gente!

**Comment by Mar0la** [comment]
Meu primeiro contato com Hacktoberfest.

**Reply by morgannadev** [reply]
E já está arrasando! Que venham muitas outras participações!

**Comment by ghost** [comment]
Esse é o terceiro ano que estou participando da Hacktoberfest e queria parabenizar a deusa @morgannadev por esse projeto, porque foi uma ideia muito legal para ajudar quem está iniciando e não possui experiência com o fluxo de trabalho do Git e GitHub, ou mesmo tem medo de contribuir por falta de experiência ou acolhimento.

Você arrasou e eu adorei criar o meu octocat, mesmo perdendo uns bons minutos indecisa do que colocar de acessório rs

**Reply by morgannadev** [reply]
Ah, são tantas opções incríveis do My Octocat que fica difícil mesmo escolher! Super obrigada por todo o apoio e por estar comigo nesse projeto, compartilhando também suas experiências nesse evento e com Open Source! Tamo junta demais!

**Comment by victorjordan95** [comment]
Esse é meu terceiro Hacktober! Participei nas ediçoes de 2018 e 2019, acabei perdendo as outras edições, mas sempre foram experiências ótimas! A interação com a comunidade e o desafio de procurar lugares para ajudar é uma sensação única! 

Adorei a forma como está sendo incentivado a criação da primeira PR, @morgannadev! Meus parabens!! <3

**Reply by morgannadev** [reply]
A comunidade sempre ajuda mesmo, né? Eu só comecei a participar por causa dela! Que bom que você curtiu participar do nosso projeto e que gostou também da ideia! Bora continuar incentivando a galera! Valeu demais!

**Comment by akiedimais** [comment]
Aprendi a usar o Git com minha amiga pra participar, estou gostando muito. <33

**Reply by SystemGuuh** [reply]
Opa comentei errado, edit: adorei amg, me diverti mto tbm

**Reply by morgannadev** [reply]
É sobre isso. A gente vai se ajudando e se divertindo enquanto aprende. Que bom que gostaram!

**Comment by SystemGuuh** [comment]
Pude participar enquanto conversava com minhas amigas, super divertido, rimos muito personalizando octocats por aqui kkkkkkkkkkk

**Reply by SystemGuuh** [reply]
adorei amg, me diverti mto tbm

**Reply by morgannadev** [reply]
É bem legal mesmo personalizar octocats nesse site! Que bom que curtiu e que bom que participou! Valeu demais por fazerem parte desse projeto, pessoal!

**Comment by lucaskja** [comment]
Fiz esse PR em uma live que a Escola de Artes, Ciências e Humanidades e foi bem legal para aprender e poder ensinar as pessoas sobre esse repositório que é uma boa porta de entrada para a HacktoberFest

**Reply by morgannadev** [reply]
Que massa saber que esse projeto foi parar até em outras lives por aí! Fico realmente muito feliz com isso! Obrigada por compartilhar e por participar, de verdade!

**Comment by cat-macedo** [comment]
Fiquei sabendo sobre a Hacktoberfest e sobre seu repositório em uma live dos veteranos do meu curso e gostei bastante de participar e criar meu octocat!

**Reply by morgannadev** [reply]
Que massa que o projeto chegou até vocês para participarem! Não sei quem divulgou em live, mas agradeço muito por terem falado desse projeto! De verdade! E muito obrigada você por ter vindo aqui participar e contribuir, @cat-macedo! Fico muito feliz com isso!

**Comment by AlexJBSilva** [comment]
Assiti a Live. Foi muito legal. Obrigado pela organização e pelo conteúdo.
Seu passo a passo de como contribuir está claro e didático.
Só consegui parar para "contribuir" com esse repo agora.
Primeira vez na Hacktoberfest.
Mais uma vez, obrigado @morgannadev !

**Reply by morgannadev** [reply]
Que massa que você curtiu o conteúdo e o projeto! E muito obrigada por ter vindo participar e contribuir também! Arrasou no PR! E parabéns por participar pela primeira vez. Que venham mais!

**Comment by MiguelPuga** [comment]
Primeira vez que estou participando do hacktoberfest, recomendação da @Anemaygi.
Muito legal participar, espero continuar contribuindo :)

**Reply by morgannadev** [reply]
Que demais saber que tiveram pessoas que te indicaram o projeto! E fico feliz que tenha sido sua primeira participação, que muitas outras venham! Obrigada por ter participado!

**Comment by leitoraincomum** [comment]
Essa é a minha quarta vez na HAcktoberfest e dessa vez de um jeito bem diferente, amei esse projeto de criar o próprio Octocat.

Obrigada @morgannadev pela iniciativa. 💜

**Reply by morgannadev** [reply]
Expert de Hacktoberfest! Obrigada demais por ter participado do projeto, Fe! E obrigada pelo feedback, fico feliz que tenha gostado do projeto!

**Reply by leitoraincomum** [reply]
Fiz um update do meu octocat, incrível como os gostos mudam com o tempo, rs

**Reply by morgannadev** [reply]
Arrasou, Fe! Obrigada por contribuir!

**Comment by Engcompaulo** [comment]
Essa é a minha segunda vez na Hacktoberfest, a ultima vez foi no ano passado e eu escolhi plantar uma árvore, dessa vez de foi mais complicado participar devido ao tempo muito corrido, mas estou na luta aqui para realizar as minhas contribuições. 

Obrigada @morgannadev pela iniciativa, um forte abraço !

**Reply by morgannadev** [reply]
Sei como é, mas fico feliz que você tenha encontrado um momento na correria para participar do nosso projeto! Valeu demais!

**Comment by GabsVentorim** [comment]
Projeto muito legal que você desenvolveu, parabéns!!!

**Reply by morgannadev** [reply]
Fico feliz que tenha curtido o projeto e que tenha participado, valeu!

**Comment by MaiaMedeiros** [comment]
Olá comunidade! 
@morgannadev Obrigada por me receber com braços abertos na comunidade dev! Espero trazer experiências em breve por aqui!

**Reply by morgannadev** [reply]
Eu que agradeço por topar participar, contribuir e compartilhar também suas experiências! Valeu demais!

**Comment by ablazejuk** [comment]
Oi! Minha segunda participação no Hacktoberfest. Achei legal a iniciativa do projeto.

**Reply by morgannadev** [reply]
Parabéns pelo segundo ano no Hacktoberfest! E obrigada pelo feedback e participação no projeto!

**Comment by priscillatrevizan** [comment]
Oi galerinha, marcando presença aqui pelo segundo ano para registrar minha mudança de estilo vibe octocat.

**Reply by morgannadev** [reply]
Maravilhosa, Pri! Valeu por estar mais um ano por aqui!

**Comment by levxyca** [comment]
Olá! Adorei criar a minha versão de Octocat e participar do projeto!

**Reply by morgannadev** [reply]
Ficou incrível, eu amei! Obrigada por participar!

**Comment by caroldonadel** [comment]
oioioi, amei esse repo, to chocada que só conheci ele agora. Muito feliz no meu quarto ano participando do hacktoberfest

**Reply by morgannadev** [reply]
Que bom que conheceu o repositório! Fico feliz que tenha gostado! E parabéns pelo quarto ano no evento!

**Comment by anamariasilvadev** [comment]
Hello World, Octocats da Comunidade! 
Muito feliz em contribuir com este projeto. É fundamental participar de iniciativas como essa para contribuir e aprender com com a comunidade tech, apoiando projetos open source na Hacktoberfest! 
Happy Hacktoberfest!

**Reply by morgannadev** [reply]
Sempre bom aprender em comunidade mesmo, concordo plenamente! Obrigada pela participação!
